tests/python/contrib/test_cmsisnn/test_networks.py

```diff
@@ -120,5 +120,32 @@ def test_cnn_small(test_runner):
     )
 
 
+@tvm.testing.requires_package("tflite")
+def test_keyword_scramble():
+    """Download keyword_scrambled and test for Relay conversion.
+    In future, this test can be extended for CMSIS-NN"""
+    # download the model
+    base_url = (
+        "https://github.com/tensorflow/tflite-micro/raw/"
+        "de8f61a074460e1fa5227d875c95aa303be01240/"
+        "tensorflow/lite/micro/models/"
```

Comment:
I think CI failed due to the extra "/" at the end 😥

Reply:
Thanks Luke 🙇